packages/fxa-settings/src/pages/Signup/container.tsx

```diff
@@ -96,6 +101,7 @@ const SignupContainer = ({
         });
         return data ? { data: { ...data, unwrapBKey } } : { data: null };
       } catch (error) {
+        // not necessarily a GraphQlError?
```

Comment:
There are two calls here. getCredentials could throw an error on crypto operations. I think it's also possible that some sort of other like a networking error could be thrown that wouldn't e a graphql error.